### PR TITLE
TShirt sizing for Deployments V1

### LINKED ISSUE #1028: Create Mechanism for applying T-Shirt Sizes to ARM Deployments
We want to support Small, Medium and Large configurations for testing. Need to implement a way to allow the size of a deployment to be selected.

### PR DESCRIPTION
**Pull request by lawrencegripper**
This PR implements the majority of the TShirt Sizing work, however, it does leave some remaining work which is flagged in the issue. 

Related #1028

### CODE REVIEW

**Comment by lawrencegripper** [comment]
@marrobi and @jjcollinge you ok to take a quick review?

**Comment by martinpeck** [comment]
BLOCKED, but ready to merge. Waiting for other stuff to be stable before we merge.

File: opencga-app/app/scripts/azure/arm/azuredeploy.json [hunk 1]
```diff
@@ -203,9 +170,131 @@
...
+                },
+                "webServers": {
+                    "minNodeCount": 1,
+                    "maxNodeCount": 2,
+                    "nodeSize": "Standard_DS2_v2"
+                }
+            },
+            {
+                "type": "1 = Medium",
+                "avereEnabled": false,
```
Comment (marrobi) [inline opencga-app/app/scripts/azure/arm/azuredeploy.json]: Should this be true?

Comment (marrobi) [reply opencga-app/app/scripts/azure/arm/azuredeploy.json]: Both azureFilesEnabled and avereEnabled are false

**Comment by lawrencegripper** [comment]
Merged into #1099 and done

**Pull request closed without merging**